internal/rbd/group/volume_group.go

```diff
@@ -151,18 +151,17 @@ func GetVolumeGroup(
 	return vg, nil
 }
 
-// String makes it easy to include the volumeGroup object in log and error
-// messages.
+// String returns the image-spec (pool/{namespace}/{name}) format of the group.
 func (vg *volumeGroup) String() string {
```

Comment:
tagging @nixpanic to verify is vg.id is not used anywhere from the Stringer call.

Reply:
it shouldn't, `vg.GetID()` is the preferred way, or in some cases `vg.id` is safe.